discovery-provider/plugins/pedalboard/apps/relay/src/txRelay.ts

```diff
@@ -26,14 +26,19 @@ export const relayTransaction = async (
   rep: FastifyReply
 ): Promise<RelayedTransaction> => {
   const requestId = uuidv4();
-  const log = (obj: unknown, msg?: string | undefined, ...args: any[]) =>
-    logger.info(obj, msg, requestId, ...args);
-  const { web3, wallets, config } = app.viewAppData();
+  const { web3, config, operatorWallet } = app.viewAppData();
   const {
     entityManagerContractAddress,
     entityManagerContractRegistryKey,
     aao,
   } = config;
+
+  const senderWallet = operatorWallet
+  const address = await senderWallet.getAddress();
```

Comment:
I think we may have an issue here when gathering the nonce, if a node can handle multiple tx submissions in parallel then transactions with the same nonce can definitely occur resulting in higher failure rates. Perhaps building in a queue into relay so they're all processed in order per node yet we don't lose visibility is the way to go?